oat-sa/extension-tao-itemqti
# Feature/AUT-208/Proper identifier validation in authoring UI

Related to: https://oat-sa.atlassian.net/browse/AUT-208

Unit test fail because of other issue - fix here https://github.com/oat-sa/extension-tao-itemqti/pull/1668/files
How to test:
- go to item Authoring
- add in `Identifier` forbidden symbol, for example`@` `=` `! `
- check warning in tooltip
- try to save item
- check warning message
- check the same errors for `Response` and `outcome Declaration`

![image](https://user-images.githubusercontent.com/25976342/112495968-6dfef100-8d95-11eb-91aa-ccf1dc3abe84.png)
![image](https://user-images.githubusercontent.com/25976342/112496036-79eab300-8d95-11eb-986f-4b3ed747aabd.png)
![image](https://user-images.githubusercontent.com/25976342/112496080-840cb180-8d95-11eb-8261-f4513eaa99fc.png)
![image](https://user-images.githubusercontent.com/25976342/117125830-b94bec80-ada2-11eb-8291-cbd72cae5591.png)
